services/slack_service.py

```diff
@@ -10,6 +10,7 @@
 
 class SlackService:
     OPERATIONS_ENGINEERING_ALERTS_CHANNEL_ID = "C033QBE511V"
+    OPS_ENG_SR_TEST_CHANNEL_ID = "C065RCW59MK"
```

Comment:
We don't need this.

Reply:
Changed this to the ops-eng-team channel as that's where we agreed the report would go